packages/effect/src/Schema.ts

```diff
@@ -6209,6 +6209,22 @@ export class DateTimeUtcFromNumber extends transformOrFail(
   }
 ).annotations({ identifier: "DateTimeUtcFromNumber" }) {}
 
+/**
+ * Defines a schema that attempts to convert a `Date` to a `DateTime.Utc` instance using the `DateTime.unsafeMake` constructor.
+ *
+ * @category DateTime.Utc transformations
+ * @since 3.11.0
+ */
+export class DateTimeUtcFromDate extends transformOrFail(
+  Date$.annotations({ description: "a date that will be parsed into a DateTime.Utc" }),
```

Comment:
I'd rather expect `DateFromSelf`

Reply:
yes, I even used `ValidDateFromSelf` because the Date must always be valid anyway